src/modules/Dropdown/Dropdown.js

```diff
@@ -460,9 +461,19 @@ export default class Dropdown extends Component {
     // open search dropdown on search query
     if (search && newQuery && !open) this.open()
 
+    // resize the search input, temporarily show the sizer so we can measure it
+    let searchWidth
+    if (newQuery) {
+      const sizer = findDOMNode(this).querySelector('.sizer')
```

Comment:
I was trying to avoid refs as much as possible.  I suppose we already have a search ref, and this is essentially the same thing.  Will update.